### PR TITLE
Tourist Website Added

### PR DESCRIPTION
**Pull request by samipak458**
### 👨‍💻 Changes proposed
Project Tourist Website Added

### ✅ Check List (Check all the applicable boxes) 

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers
It is  fully responsive having beautiful animation. Have a look my website: [Tourist](https://merapakistan.samimunir2002.repl.co/)

### 📷 Screenshots
image

### CODE REVIEW

**Comment by samipak458** [comment]
@Astrodevil kindly review my PR. Thank you.

**Review by Astrodevil — CHANGES_REQUESTED** [review]
Please move img and gif files from css to respective folder. Make required changes.

File: cards.json [hunk 1]
```diff
@@ -724,5 +724,10 @@
         "name": "YouTube Clone",
         "image": "assets/Images/YouTubeClone/YouTubeClone.png",
         "link": "Public/YouTube_Clone/index.html"
+    },
+    {
+        "name": "Tourist Website",
```
Comment (Astrodevil) [inline cards.json:729]: Please update cards.json alphabetically

Comment (samipak458) [reply cards.json]: Done

**Comment by samipak458** [comment]
All the required has been made. Kindly review and merge my PR. Thank you.

**Comment by Astrodevil** [comment]
You can create new PR properly.

**Comment by samipak458** [comment]
@Astrodevil here the new PR link: https://github.com/ZeroOctave/ZeroOctave-Javascript-Projects/pull/756 kindly review it.

**Pull request closed without merging**